platform/security/servlet/security-servlet-whoami/src/main/java/org/codice/ddf/security/servlet/whoami/WhoAmI.java

```diff
@@ -202,4 +203,8 @@ public boolean isGuest() {
       return claims;
     }
   }
+
+  private static boolean attributeValuesAreNonNull(Attribute attribute) {
+    return !attribute.getValues().stream().allMatch(Objects::isNull);
```

Comment:
✏️ `attribute.getValues().stream().noneMatch(Objects::isNull);`

Reply:
I want to return false when *every* element in that list is null, whereas your suggestion will return false when *any* element in the list is null.

But now that I think about it, I wonder if having *any* nulls in that list can cause a NPE.

After some more thinking, the NPE situation definitely sounds like it could happen. I'll go ahead and change the `SubjectUtils` logic to look like this as well.